package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bellese/angular-design-system",
-  "version": "4.0.9",
+  "version": "4.4.0",
```

Comment:
could you also undo this change too if you don't mind? I think this was probably a downstream effect from running npm build with the same change in the `package.json` file

Reply:
Done! Yeah missed that b/c of how GH collapses this file - which normally I wouldn't mind if the change wasn't a 1-liner 🤷